docs/en/Running-Alluxio-On-Kubernetes.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: global
+title: Running Alluxio on Kubernetes
+nickname: Alluxio on Kubernetes
+group: Deploying Alluxio
+priority: 3
+---
+
+Alluxio can be run on Kubernetes. This guide demonstrates how to run Alluxio
+on Kubernetes using the specification that comes in the Alluxio Github repository.
+
+# Basic Tutorial
+
+This tutorial walks through a basic Alluxio setup on Kubernetes.
+
+## Prerequisites
+
+- A Kubernetes cluster (version >= 1.8). Alluxio workers will use `emptyDir` volumes with a 
+restricted size using the `sizeLimit` parameter. This is an alpha feature in Kubernetes 1.8.
+Please ensure the feature is enabled.
+- An Alluxio docker image. Refer to [this page](Running-Alluxio-On-Docker.html) for instructions
+to build an image.
+
+## Clone the Alluxio repo
+
+```bash
+$ git clone https://github.com/Alluxio/alluxio.git
+$ cd integration/kubernetes
+```
+
+The kubernetes specifications required to deploy Alluxio can be found under `integration/kubernetes`.
+
+## Short-circuit operations
+
+Short-circuit access enable clients to perform read and write operations directly against the 
```

Comment:
we should make it clear at the start that this is optional, and describe when/why users would want to use domain socket for short-circuit operations

Reply:
i don't think it is optional for anything except a prototype. are you suggesting instructions for the alternative of disabling short circuit all together? (i guess not)

Reply:
It's unnecessary in scenarios where the compute is remote from the Alluxio workers. This isn't the common case, but it isn't rare either.

my concern is that there may be situations where the person setting up Alluxio isn't able to set up the domain socket for whatever reason. If good local performance isn't important, setting up the domain socket doesn't need to be a blocker for them

Reply:
ack